src/main/kotlin/by/architecture_map/belarus/service/impl/ArchitecturalStyleServiceImpl.kt

```diff
@@ -0,0 +1,76 @@
+package by.architecture_map.belarus.service.impl
+
+import by.architecture_map.belarus.entity.ArchitecturalStyle
+import by.architecture_map.belarus.repository.ArchitecturalStyleRepository
+import by.architecture_map.belarus.repository.ImageRepository
+import by.architecture_map.belarus.service.ArchitecturalStyleService
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.stereotype.Service
+
+@Service
+class ArchitecturalStyleServiceImpl(
+        private val architecturalStyleRepository: ArchitecturalStyleRepository,
+        private val imageRepository: ImageRepository
```

Comment:
implement a separate `ImageService` and work with images using it in dofferent services.
the most common use case is one service should work with one repo